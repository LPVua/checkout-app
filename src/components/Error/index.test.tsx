import * as React from 'react'
import {render, cleanup} from '@testing-library/react';
import { Error } from '.';

describe('Error', () => {
  afterEach(cleanup);

  it('should render proper markup', () => {
    const { asFragment } = render(<Error>Message</Error>)
    
    expect(asFragment()).toMatchSnapshot();
  })
});
