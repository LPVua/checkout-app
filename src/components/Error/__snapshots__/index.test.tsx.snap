// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render proper markup 1`] = `
<DocumentFragment>
  .emotion-0 {
  color: var(--red-1);
}

<div
    class="emotion-0"
  >
    Message
  </div>
</DocumentFragment>
`;
